package.json

```diff
@@ -36,6 +36,7 @@
     "react-redux": "^4.4.1",
     "redux": "^3.3.1",
     "redux-thunk": "^2.0.1",
+    "sparql-client-2": "^0.6.0",
```

Comment:
Also why using this instead of what Fabio recommended (https://github.com/thomasfr/node-sparql-client) in his lectures ? I'm sure you have a reason X) but I couldn't see it immedietely